aten/src/THCUNN/ClassNLLCriterion.cu

```diff
@@ -192,3 +192,6 @@ __global__ void cunn_ClassNLLCriterion_updateGradInput_kernel(
 
 #include <THCUNN/generic/ClassNLLCriterion.cu>
 #include <THC/THCGenerateFloatTypes.h>
+
+#include <THCUNN/generic/ClassNLLCriterion.cu>
+#include <THC/THCGenerateBFloat16Type.h>
```

Comment:
I'm not very happy about this (adding features to TH library, correct order of operations is to port and then add functionality), but given that `THCGenerateBFloat16Type.h` is an existing header I'll give you a pass.